.github/workflows/pr_label_check.yml

```diff
@@ -13,24 +16,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check aspect label
-        uses: sugarshin/required-labels-action@v0.3.2
+        uses: agilepathway/label-checker@v1.2.9
         with:
-          required_oneof: "💻 aspect: code,📄 aspect: text,🤖 aspect: dx,🕹 aspect: interface,♿️ aspect: a11y"
+          any_of: api,bug,build,dependencies,documentation,enhancement,no-changelog,refactoring
```

Comment:
`any_of` is correct but please keep the old list of labels.

Reply:
The new labels have been removed and old labels have been restored.